src/lib/y2partitioner/actions/controllers/filesystem.rb

```diff
@@ -184,6 +188,19 @@ def partition_id
           blk_device.is?(:partition) ? blk_device.id : nil
         end
 
+        def encrypt_types
+          types = [Y2Storage::EncryptionType::LUKS1]
+
+          # FIXME: PLAIN instead of TWOFISH (TWOFISH only for testing purposes)
+          types << Y2Storage::EncryptionType::TWOFISH if swap?
```

Comment:
Replace with the type provided by the library for plain encryption: I guess something like `::PLAIN`.